Cartfile.private

```diff
@@ -1 +1,2 @@
-github "Quick/Quick" ~> 1.1
+github "Quick/Quick" ~> 2.0
+github "AliSoftware/OHHTTPStubs"
```

Comment:
I know this is not you @nonec but if you run `carthage update` now, it will break current implementation of tests as this dependency dropped `OH` prefix from all its class names. All it will bring is a headache. :|

Reply:
That is a headache, yes :|